### ISSUE TITLE
Hearthfire Extended - do not clean ITM

### ISSUE THREAD

**Issue by Nebulous112** [loot/skyrim#104]
Hello,

I am using Hearthfire Extended (the "All DLCs - Legendary Edition" version), v1.03. The plugin name is: hearthfireextended.esp

The mod author states on the Nexus description page ( http://www.nexusmods.com/skyrim/mods/66783/? ): 
"Cleaned with TES5EDIT. No bad records. No further cleaning required.
Relevant fixes from USKP forwarded. USKP is however not a requirement.
(While LOOT recognises that the mod is clean, cleaning with tes5edit actually does remove one record. This record is required for steward dialogue to function correctly, so I don't recommend you do this.)"

This information is slightly incorrect, as LOOT does in fact show one ITM, and recommends cleaning in TES5Edit.

I suggest removing the ITM warning, and perhaps putting a warning in to not clean the mod.

I am very new to GitHub (and I have zero coding experience), however I did take a look at making a pull request for this. However, looking at the masterlist, I noticed two ITMs listed for hearthfireextended.esp, and this confused me. I assume this is due to a difference between file versions, or something.

Anyway, I figure I will probably screw something up if I tried to do this, so I had better leave it to the pros! ;-)

Thanks again, Wrinkly and team for all you guys do!

Cheers,

Nebulous

**Comment by Freso** [comment]
I think you should give changing it yourself a go. :) Us "pros" will then be able to provide feedback on what you've done and soon you'll be a pro yourself! :D

**Comment by Kinaga** [comment]
The issue here is whether ITMs in DialogueViews should be considered to make a mod "dirty", since I don't think the game engine reads DialogueViews at all. At the time I wrote the quoted line I suspected that I had edited the DialogueView in a way that TES5Edit did not recognise, however I now think that the DialogueView is used only by the creation kit itself.

I intend to remove this edit from the mod in its next update. It might be a good idea to set up a rule for LOOT to ignore ITMs in DialogueViews since they are not used by the game anyway so cannot cause problems (assuming this information is correct).

**Comment by Nebulous112** [comment]
@Kinaga Thanks for the additional info. :-) Please correct me if I'm wrong...what I understand (in layman's terms) from your reply is that leaving the ITM will not hurt anything, but cleaning it will not hurt anything either. If you plan to remove this edit in the next update, would you like me to close this ticket?

@Freso If I had known earlier that you folks would be so willing to help a newbie learn, I definitely would have tried! :-) However, with this particular issue, I am unsure whether it is a waste of time to proceed (as per above). When I next come across a valid masterlist issue, I will definitely give it a go!

Thanks to all for your time.

**Comment by Kinaga** [comment]
The issue related to this mod can be ignored as I will be removing the ITM in the next update. This is probably grounds to close the issue report.

The issue about whether LOOT should show warnings for ITMs in DialogueViews at all, since the game does not use that record (AFAIK), remains.

**Comment by Freso** [comment]
Personally, I'd say "better safe than sorry" and report that there's an ITM. ITMs aren't critical to begin with: unlike UDRs, they shouldn't cause the game to crash (in far most cases), but are more likely to just give some random glitches (e.g., adjusted values not being aligned, some messages in English, some in your local language, etc.). So unless there's a specific reason to *not* clean an ITM (e.g., because you have adjusted values and *need* some of the values to be vanilla to work well with the others), I'd say we report it.

**Comment by Nebulous112** [comment]
Closing this issue as requested.
